io.github.RobertBeckebans.RBDoom3BFG.GL.yaml

```diff
@@ -0,0 +1,64 @@
+app-id: io.github.RobertBeckebans.RBDoom3BFG.GL
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: rbdoom3bfg.sh
+finish-args:
+  - --device=all
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+cleanup:
+  - /include
+  - '*.a'
+  - '*.la'
+  - /lib/pkgconfig
+  - /share/man
+  - /share/cmake
+modules:
+
+  - shared-modules/glu/glu-9.json
+
+  # Build application from Git sources
+  - name: rbdoom3bfg-build
+    buildsystem: cmake
+    builddir: true
+    subdir: neo
+    no-make-install: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DSDL2=ON
+      - -DFFMPEG=OFF
+      - -DBINKDEC=ON
+    sources:
+      - type: git
+        url: https://github.com/RobertBeckebans/RBDOOM-3-BFG
+        tag: v1.4.0
+    post-install:
+      # Install executable
+      - install -Dm 755 RBDoom3BFG /app/bin/RBDoom3BFG
+      # Install base files
+      - mkdir -p /app/share/rbdoom3bfg/base
+      - cp --recursive ../../base/* --target-directory /app/share/rbdoom3bfg/base
+
+  - name: rbdoom3bfg-desktop
```

Comment:
this shild  be in the main module.

Reply:
Keeping the desktop module separated from source is for avoiding to recompile the source (time consuming) when something must be changed only in app files. Basically it helps development. But I guess the suggested approach improves delivery efficiency.